packages/wonder-blocks-data/src/util/types.js

```diff
@@ -15,12 +15,12 @@ export type Result<TData: ValidData> =
 
 export type CacheEntry<TData: ValidData> =
     | {|
-          error: string,
-          data?: ?void,
+          +error: string,
+          +data?: ?void,
       |}
     | {|
-          data: TData,
-          error?: ?void,
+          +data: TData,
+          +error?: ?void,
```

Comment:
Flow was outputting some new errors about these being read-only in some cases and not others, so I just marked them as read-only since that's right anyway.